ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/router-bgp.md

```diff
@@ -1338,6 +1346,9 @@
         redistribute_routes:
           - source_protocol: <str; required; unique>
             route_map: <str>
+
+            # RCF function name without parenthesis.
```

Comment:
can you confirm again that rcf is supported for address_family_ipv4_multicast->redistribute_routes as am not able to get in device
```
wan1-dc1(config-router-bgp)#address-family ipv4 multicast 
wan1-dc1(config-router-bgp-af)#redistribute connected ?
  route-map  Name a route map
  <cr>       

wan1-dc1(config-router-bgp-af)#redistribute connected 
```